docs/reference-manual/src/main/jbake/content/start-domain.adoc

```diff
@@ -81,14 +81,29 @@ asadmin-options::
   Specifies whether the domain is started with
   http://java.sun.com/javase/technologies/core/toolsapis/jpda/[Java
   Platform Debugger Architecture (JPDA)]
-  (http://www.oracle.com/technetwork/java/javase/tech/jpda-141715.html)
-  debugging enabled. +
+  (https://docs.oracle.com/en/java/javase/14/docs/specs/jpda/conninv.html)
```

Comment:
I'd refer to LTS java version instead of the "latest available version"

Reply:
Isn't LTS just something a random vendor or distributor attaches to a certain build? LTS is not inherently attached to Java itself, is it?

Reply:
While you might be right according to [JEP 322](https://openjdk.java.net/jeps/322), is there any vendor offering LTS for JDK 14? Is there 100% guarantee that what the doc you are referring to describes can be used on the JDK officially supported by GF, which is supposed to be JDK 11 (hint: `allow` is not supported on JDK11)?

Reply:
It's more intended as a general reference to JDPA as the previous link was broken, but the version supported by GF is a good default. I'll change it to that, thanks!